geotrek/signage/templates/signage/blade_detail.html

```diff
@@ -60,7 +60,7 @@ <h3>{% trans "Lines" %}</h3>
                 <tr>
                     <td>{{ line }}</td>
                     {% if is_direction_on_lines_enabled %}<td>{{ line.direction }}</td>{% endif %}
-                    <td>{{ line.text }}</td>
+                    <td>{{ line.text|default_if_none:""  }}</td>
```

Comment:
This is not necessary anymore now that field is `null=False, default=""`

Reply:
Done, thank you :)